app/views/profiles/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+  <div class="medium-4 columns">
+    <div>
+      <%= render "profile" %>
+    </div>
+  </div>
+  <div class="medium-4 columns">
```

Comment:
I'd maybe suggest `large-4 medium-6` on the profile and password sections, and just `large-4` on the key section, since things get pretty squished at medium res with three columns. That'd make things wrap a bit better.